src/css/_uswds_variables.scss

```diff
@@ -0,0 +1,153 @@
+
+$color-white:                #ffffff !default;
+$color-black:                #000000 !default;
+$color-black-light:          #212121 !default;
+
+$color-gray-dark:            #323a45 !default;
+$color-gray:                 #5b616b !default;
+$color-gray-medium:          #757575 !default;
+$color-gray-light:           #aeb0b5 !default;
+$color-gray-lighter:         #d6d7d9 !default;
+$color-gray-lightest:        #f1f1f1 !default;
+
+$color-gray-warm-dark:       #494440 !default;
+$color-gray-warm-light:      #e4e2e0 !default;
+$color-gray-cool-light:      #dce4ef !default;
+
+$color-gold:                 #fdb81e !default;
+$color-gold-light:           #f9c642 !default;
+$color-gold-lighter:         #fad980 !default;
+$color-gold-lightest:        #fff1d2 !default;
+
+$color-green:                #2e8540 !default;
+$color-green-light:          #4aa564 !default;
+$color-green-lighter:        #94bfa2 !default;
+$color-green-lightest:       #e7f4e4 !default;
+
+$color-cool-blue:            #205493 !default;
+$color-cool-blue-light:      #4773aa !default;
+$color-cool-blue-lighter:    #8ba6ca !default;
+$color-cool-blue-lightest:   #dce4ef !default;
+
+$color-purple:               #4c2c92 !default;
+
+// Functional colors
+$color-primary:              $color-blue !default;
+$color-primary-darker:       $color-blue-darker !default;
+$color-primary-darkest:      $color-blue-darkest !default;
+
+$color-primary-alt:          $color-aqua !default;
+$color-primary-alt-dark:     $color-aqua-dark !default;
+$color-primary-alt-darkest:  $color-aqua-darkest !default;
+$color-primary-alt-light:    $color-aqua-light !default;
+$color-primary-alt-lightest: $color-aqua-lightest !default;
+
+$color-secondary:            $color-red !default;
+$color-secondary-dark:       $color-red-dark !default;
+$color-secondary-darkest:    $color-red-darkest !default;
+$color-secondary-light:      $color-red-light !default;
+$color-secondary-lightest:   $color-red-lightest !default;
+
+$color-base:                 $color-black-light !default;
+$color-focus:                $color-gray-light !default;
+$color-visited:              $color-purple !default;
+
+$color-shadow:               rgba(#000, 0.3) !default;
+$color-transparent:          rgba(#000, 0) !default;
+
+// Mobile First Breakpoints
+$small-screen:  481px !default;
+$medium-screen: 600px !default;
+$large-screen:  1201px !default;
+
+// Grid column counts by screen size
+$grid-columns-small: 1 !default;
+$grid-columns-medium: 6 !default;
+$grid-columns-large: 12 !default;
+
+// @media single-keyword helpers
+$small: new-breakpoint(min-width $small-screen $grid-columns-small) !default;
+$medium: new-breakpoint(min-width $medium-screen $grid-columns-medium) !default;
+$large: new-breakpoint(min-width $large-screen $grid-columns-large) !default;
+
+// Relative font and image file paths
+$font-path:   '../../node_modules/uswds/src/fonts' !default;
+$image-path:  '../../node_modules/uswds/src/img' !default;
```

Comment:
Updated these two lines (122-123). We're pointing back to the node_module folder for USWDS's fonts and images. For some reason, relative paths where the only way this worked.

Reply:
I can update this to work with full paths in a later branch

Reply:
Or actually you can update this by adding the --include-path flag to the package.json in the build-css and watch-css steps as described here: 
https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#adding-a-css-preprocessor-sass-less-etc
cmd-f for include-path and you'll see it